plugins/field-date/test/test.mocha.js

```diff
@@ -0,0 +1,159 @@
+/**
+ * @license
+ * Copyright 2019 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const assert = require('assert');
+require('blockly');
```

Comment:
Why is this needed? and if it is, then maybe field_slider should have it too?

Reply:
Not needed, removed.